openmc/universe.py

```diff
@@ -111,6 +111,17 @@ def create_xml_subelement(self, xml_element, memo=None):
 
         """
 
+    @abstractmethod
+    def _partial_deepcopy(self):
+        """Deepcopy all parameters of an openmc.UniverseBase object except its cells.
+        This should only be used from the openmc.UniverseBase.clone() context.
+
+        Returns
+        -------
+        None
+
```

Comment:
For methods that don't have a return value, the going practice is to just omit the "Returns" section.
```suggestion
```